File: API/Models/Account.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace API.Models
{
    [Table("Account")]
    public partial class Account
    {
        public Account()
        {
            AccountRoles = new HashSet<AccountRole>();
            AccountFunctions = new HashSet<AccountFunction>();
        }

        [Key]
        public long Id { get; set; }
        [Required]
        public string UserName { get; set; }
        [Required]
        public string Password { get; set; }
        public bool? Status { get; set; }
        public bool? IsDelete { get; set; }
        public long CreateBy { get; set; }
        [Column(TypeName = "datetime")]
        public DateTime CreateTime { get; set; }
        public long? UpdateBy { get; set; }
        [Column(TypeName = "datetime")]
        public DateTime? UpdateTime { get; set; }
        public long AccountTypeId { get; set; }
        [ForeignKey("AccountTypeId")]
        [InverseProperty("Accounts")]
        public virtual AccountType AccountType { get; set; }
        [InverseProperty("Account")]
        public virtual ICollection<AccountRole> AccountRoles { get; set; }
        [InverseProperty("Account")]
        public virtual ICollection<AccountFunction> AccountFunctions { get; set; }
    }
}

File: SPA/src/app/core/models/admins/account.ts
export interface Account {
    id: number;
    userName: string;
    password: string;
    status: boolean | null;
    isDelete: boolean | null;
    createBy: string;
    createTime: string | Date;
    updateBy: string;
    updateTime: string | null | Date;
    accountTypeId: string;
    roleIds: string[];
    functionIds: string[];
}

export interface AccountChangePassword {
    id: number;
    password: string;
    newPassword: string;
    confirmPassword: string;
    updateBy: string;
    updateTime: string | null | Date;
}